features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha256_alt_sw.c

```diff
@@ -186,24 +178,29 @@ void mbedtls_sha256_sw_process( mbedtls_sha256_sw_context *ctx, const unsigned c
         A[i] = ctx->state[i];
 
 #if defined(MBEDTLS_SHA256_SMALLER)
-    for( i = 0; i < 64; i++ )
-    {
+    for( i = 0; i < 64; i++ ) {
```

Comment:
Consider using `#define` for `64`, `16`, and `8` in this file, where it makes sense to do so, for readability's sake.

Reply:
@Patater This `sha256_alt_sw.c` file is cloned from `mbed-os/features/mbedtls/src/sha256.c` for fallback. So I don't make other change to it.